core/templates/dev/head/components/exploration_summary_tile_directive.html

```diff
@@ -4,6 +4,41 @@
       <div class="top-section" style="background-color: <[getThumbnailBgColor()]>;">
         <img class="thumbnail-image" ng-src="<[getThumbnailIconUrl()]>">
         <h2 class="exploration-title protractor-test-exp-summary-tile-title"><[getExplorationTitle()]></h2>
+
+        <div class="exploration-summary-contributors" ng-if="contributors.length > 0"
+             ng-mouseenter="$parent.allContributorsAreVisible = true"
```

Comment:
We discourage the use of $parent. Can you bind ng-mouseenter and ng-mouseleave to methods in the controller, instead?